doc/api/http.md

```diff
@@ -2247,6 +2304,15 @@ response
   .end(body);
 ```
 
+`headers` may also be an instance of the [`Headers`][].
+
+```js
+const body = 'hello world';
+response
+  .writeHead(200, new Headers({ foo: 'bar' }))
```

Comment:
hard no to this.

Reply:
Couls you explain why it is a bad idea?

Reply:
The whole point of adding setHeaders is avoiding to add more logic to writeHead()

Reply:
Ok I understand I got carried away by the enthusiams 😅 I'll revert these changed and leave only `res.setHeaders(headers)`